### PR TITLE
Update vcf2db dependencies

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/vcf2db/tests/nextflow.config
Comment (maxulysse) [inline modules/nf-core/vcf2db/tests/nextflow.config:1]: is this actually useful?

**Comment by matthdsm** [comment]
can you first clean up your git history 😅

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
> can you first clean up your git history 😅

doesn't matter it'll be squashed

**Comment by maxulysse** [comment]
Updating the branch because you have some unrelated tests coming in

**Comment by maxulysse** [comment]
> > can you first clean up your git history 😅
> 
> doesn't matter it'll be squashed

but yeah, what happened there?

File: modules/nf-core/vcf2db/tests/nextflow.config
Comment (nvnieuwk) [reply modules/nf-core/vcf2db/tests/nextflow.config:1]: Nope it came with the command :p, I'll remove it!

**Comment by nvnieuwk** [comment]
> > > can you first clean up your git history 😅
> > 
> > 
> > doesn't matter it'll be squashed
> 
> but yeah, what happened there?

Don't ask me lol, it's been like that for a while, I'll fix it when I have merged all branches that are in development :p

**Comment by nvnieuwk** [comment]
Singularity tests are failing because the image has not yet been uploaded to the repo

**Pull request merged**